pages/[...pages].tsx

```diff
@@ -32,11 +37,22 @@ export async function getStaticProps({
   }
 }
 
-export async function getStaticPaths() {
+export async function getStaticPaths({ locales }: GetStaticPathsContext) {
   const { pages } = await getAllPages()
+  const [invalidPaths, log] = missingLocaleInPages()
+  const paths = pages
+    .map((page) => page.url)
+    .filter((url) => {
+      if (!url || !locales) return url
```

Comment:
is this line correct?

Reply:
If there's no `url`, it returns `url` and the page will be ignored (note this is happening inside `.filter()`).

If there's no `locale`, it returns the `url` of the page, the example works if i18n is disabled (I tested that multiple times 👍 )

Reply:
ok!